### PR TITLE
Add Cargo.toml fields necessary to be published on crates.io

### LINKED ISSUE #40: Prepare dusk-pki to be published on crates.io

### PR DESCRIPTION
**Pull request by ZER0**
- Add a short description to README.md

Resolves: #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# Dusk Public Key Infrastructure
+# dusk-pki
+
+![Build Status](https://travis-ci.com/dusk-network/dusk-pki.svg?branch=master)](https://travis-ci.com/dusk-network/dusk-pki)
+![Repository](https://dusk-network.github.io/dusk-pki/repo-badge.svg)](https://github.com/dusk-network/dusk-pki)
+![Documentation](https://dusk-network.github.io/dusk-pki/badge.svg)](https://dusk-network.github.io/dusk-pki/index.html)
```
Comment (vlopes11) [inline README.md]: The documentation should be `[Documentation](https://docs.rs/dusk-pki/)`

**Review by vlopes11 — APPROVED** [review]
LGTM, small nit

File: Cargo.toml [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+version = "0.6.1"
 authors = ["zer0 <matteo@dusk.network>", "Victor Lopez <victor@dusk.network>"]
 edition = "2018"
 
+repository = "https://github.com/dusk-network/dusk-pki"
+keywords = ["cryptography", "crypto", "pki", "bls12-381"]
+categories =["cryptography", "cryptography::cryptocurrencies", "no-std", "wasm"]
+description = "A library that holds the types and functions required to perform keys operations in Dusk"
+license = "MPL-2.0"
+
```
Comment (CPerezz) [inline Cargo.toml:13]: Should we exclude the .github and other similar useless files for the package???

**Review by CPerezz — APPROVED** [review]
Just a suggestion, otherwais LGTM

**Pull request merged**